src/model/document.js

```diff
@@ -341,48 +425,30 @@ export default class Document {
 	}
 
 	/**
-	 * Fired when document changes by applying an operation.
-	 *
-	 * There are several types of change:
+	 * Performs post-fixer loops. Executes post-fixer callbacks as long as neither of them has done any changes to model.
 	 *
-	 * * 'insert' when nodes are inserted,
-	 * * 'remove' when nodes are removed,
-	 * * 'reinsert' when remove is undone,
-	 * * 'move' when nodes are moved,
-	 * * 'rename' when element is renamed,
-	 * * 'marker' when a marker changes (added, removed or its range is changed),
-	 * * 'addAttribute' when attributes are added,
-	 * * 'removeAttribute' when attributes are removed,
-	 * * 'changeAttribute' when attributes change,
-	 * * 'addRootAttribute' when attribute for root is added,
-	 * * 'removeRootAttribute' when attribute for root is removed,
-	 * * 'changeRootAttribute' when attribute for root changes.
-	 *
-	 * @event change
-	 * @param {String} type Change type.
-	 * @param {Object} data Additional information about the change.
-	 * @param {module:engine/model/range~Range} [data.range] Range in model containing changed nodes. Note that the range state is
-	 * after changes has been done, i.e. for 'remove' the range will be in the {@link #graveyard graveyard root}.
-	 * The range is not defined for root, rename and marker types.
-	 * @param {module:engine/model/position~Position} [data.sourcePosition] Change source position.
-	 * Exists for 'remove', 'reinsert' and 'move'.
-	 * Note that this position state is before changes has been done, i.e. for 'reinsert' the source position will be in the
-	 * {@link #graveyard graveyard root}.
-	 * @param {String} [data.key] Only for attribute types. Key of changed / inserted / removed attribute.
-	 * @param {*} [data.oldValue] Only for 'removeAttribute', 'removeRootAttribute', 'changeAttribute' or
-	 * 'changeRootAttribute' type.
-	 * @param {*} [data.newValue] Only for 'addAttribute', 'addRootAttribute', 'changeAttribute' or
-	 * 'changeRootAttribute' type.
-	 * @param {module:engine/model/rootelement~RootElement} [data.root] Root element which attributes got changed. This is defined
-	 * only for root types.
-	 * @param {module:engine/model/batch~Batch} batch A {@link module:engine/model/batch~Batch batch}
-	 * of changes which this change is a part of.
+	 * @private
 	 */
+	_callPostFixers( writer ) {
+		let wasFixed = false;
+
+		do {
+			for ( const callback of this._postFixers ) {
+				wasFixed = callback( writer );
+
+				if ( wasFixed ) {
+					break;
+				}
+			}
+		} while ( wasFixed );
+	}
 
 	/**
-	 * Fired when all queued document changes are done. See {@link module:engine/model/model~Model#change}.
+	 * Fired after outermost {@link module:engine/model/model~Model#change change} or
+	 * {@link module:engine/model/model~Model#enqueueChange enqueueChange} block has been executed and
```

Comment:
That's not true about the outermost `enqueueChange`. The change event will be fired after each enqueueChange block.